charts/ccd-definition-designer-api/values.template.yaml

```diff
@@ -0,0 +1,43 @@
+postgresql:
+  initdbScripts:
+    init.sql: |-
+      CREATE USER ccd WITH PASSWORD 'ccd';
+      CREATE DATABASE ccd_definition_designer WITH OWNER = ccd ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  nameOverride: postgres
+  persistence:
+    enabled: false
+
+java:
+  image: ${IMAGE_NAME}
+  ingressHost: ${SERVICE_FQDN}
+  applicationPort: 4454
+  environment:
+    DEFINITION_DESIGNER_DB_HOST: ${SERVICE_NAME}-postgres
+    DEFINITION_DESIGNER_DB_PORT: 5432
+    DEFINITION_DESIGNER_DB_NAME: ccd_definition_designer
+    DEFINITION_DESIGNER_DB_USERNAME: ccd
+    DEFINITION_DESIGNER_DB_PASSWORD: ccd
+    DEFINITION_DESIGNER_DB_USE_SSL: false
+    ENABLE_DB_MIGRATE: true
+
+    IDAM_USER_URL: http://betaDevBccidamAppLB.reform.hmcts.net
+    DEFINITION_STORE_IDAM_KEY: ${CCD_DEFINITION_SERVICE_SECRET}
+
+    DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,jui_webapp,pui_webapp
+
+    #AZURE_STORAGE_CONNECTION_STRING = "${data.azurerm_key_vault_secret.storageaccount_primary_connection_string.value}"
+    #AZURE_STORAGE_BLOB_CONTAINER_REFERENCE = "${azurerm_storage_container.imports_container.name}"
+    AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED: true
```

Comment:
We do not need `AZURE_STORAGE_DEFINITION_UPLOAD_ENABLED`  I believe

Reply:
Probably not, but again it needs to be removed from all the infrastructure then.